homeassistant/components/tod/binary_sensor.py

```diff
@@ -174,6 +174,14 @@ def _calculate_initial_boudary_time(self):
 
         self._time_before = before_event_date
 
+        # Check after value is today
```

Comment:
Please make this comment a bit more descriptive, so that everyone can understand it right away.